extensions/wikia/ARecoveryEngine/js/adBlockDetection.js

```diff
@@ -0,0 +1,57 @@
+/*global define*/
+define('ext.wikia.aRecoveryEngine.adBlockDetection', [
```

Comment:
the point was to create an interface for all detection no matter what's underneath. For now it's SP only, but it can be anything and the point of this module is just to deliver information if user is blocking ads, no matter who is detecting that